config.yaml

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
This is a json, not a yaml.
Also, you are passing this through a config map, so why is this here?

Reply:
that's where I originally developed it, it doesn't use it anymore